src/ray/core_worker/common.h

```diff
@@ -64,23 +64,23 @@ class RayFunction {
 
 /// Options for all tasks (actor and non-actor) except for actor creation.
 struct TaskOptions {
-  TaskOptions() {}
-  TaskOptions(std::string name,
-              int num_returns,
-              std::unordered_map<std::string, double> &resources,
-              const std::string &concurrency_group_name = "",
-              int64_t generator_backpressure_num_objects = -1,
-              const std::string &serialized_runtime_env_info = "{}",
-              bool enable_task_events = kDefaultTaskEventEnabled,
-              const std::unordered_map<std::string, std::string> &labels = {})
-      : name(name),
-        num_returns(num_returns),
-        resources(resources),
-        concurrency_group_name(concurrency_group_name),
-        serialized_runtime_env_info(serialized_runtime_env_info),
-        generator_backpressure_num_objects(generator_backpressure_num_objects),
-        enable_task_events(enable_task_events),
-        labels(labels) {}
+  TaskOptions() = default;
+  TaskOptions(std::string name_p,
+              int num_returns_p,
```

Comment:
what does `_p` mean?

Reply:
parameter

Reply:
Usually people don't need to worry about it, since google coding style suggests `_` suffix for data members

Reply:
But for struct, its suggests no suffix or prefix.
The `_p` coding practice is commonly used in db area (from what I know), like db written in C doesn't have private/public accessor, duckdb encourages no prefix/suffix for data members.
Example for duckdb: https://github.com/duckdb/duckdb/blob/9293db6aef8b7da7dfd8adc6140f7a8dc08cee0b/src/execution/operator/persistent/physical_insert.cpp#L25-L32